tests/e2e/support/objects/app-admin-settings/general/actions.ts

```diff
@@ -0,0 +1,16 @@
+export const uploadLogo = async (path, page): Promise<void> => {
+  await page.click('#logo-context-btn')
+
+  const logoInput = await page.$('#logo-upload-input')
+  await logoInput.setInputFiles(path)
+
+  await page.waitForSelector('text=Logo was uploaded successfully')
+}
+
+export const resetLogo = async (page): Promise<void> => {
+  await page.click('#logo-context-btn')
+
+  await page.click('.oc-general-actions-reset-logo-trigger')
+
+  await page.waitForSelector('text=Logo was reset successfully. Reloading page...')
```

Comment:
Please avoid checks on text if possible. What about waiting until the page is reloaded and then check the img src? The way it's implemented right now has the downside of not knowing if the reset really worked or it just printed the success message (as a false positive).